e2e/tests/integrations/integrator/add-organizer-uitpas-integration.test.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect, Page } from "@playwright/test";
+import { IntegrationType } from "@app-types/IntegrationType";
+import { createIntegrationAsIntegrator } from "./create-integration.js";
+import { requestActivationAsIntegrator } from "./request-activation.js";
+
+export async function addUiTPASOrganizer(
+  page: Page,
+  organizerName = "publiq vzw - UiTPAS organizer"
+) {
+  await page.getByRole("button", { name: "Organisaties" }).click();
+  await page.getByRole("button", { name: "Organisatie toevoegen" }).click();
+  await page.getByRole("textbox").click();
+  await page.getByRole("textbox").fill(organizerName);
+  await page.locator("li").filter({ hasText: organizerName }).click();
+  await page.getByRole("button", { name: "Bevestigen" }).click();
```

Comment:
Since you use this function in multiple places, you can add an assertion here that the action of adding an organizer was succesfull